File: Python/PRAK605-2210817120009-Najah_Maisyaroh.py
baris = int(input())
matrix = []
a = []
b = []
print("Matriks A")
for i in range(baris):
      matriks1 = [int (j) for j in input().split()]
      a.append(matriks1)
      matrix.append(a)
print("Matriks B")
for i in range(baris):
      matriks2 = [int (j) for j in input().split()]
      b.append(matriks2)
      matrix.append(b)
hasil=[[0 for i in range(baris)] for j in range(baris)]
for i in range(baris):
	for j in range(baris):
		for k in range(baris):
			hasil[i][j]=hasil[i][j]+a[i][k] * b[k][j]
print("Matriks AXB")
for i in range(baris):
      for j in range (baris):
            print(format(hasil[i][j]), end=" ")
      print()

File: C/PRAK605-2210817120009-Najah_Maisyaroh.c
#include <stdio.h>
int main(){
    int bil1[50][50], bil2[50][50], bil3[50][50];
    int i, j, c, d, hasil=0;
    scanf("%d", &d);
    printf("Matriks A\n");
    for(i=0;i<d;i++){
        for(j=0;j<d;j++){
            scanf("%d", &bil1[i][j]);}}
    printf("Matriks B\n");
    for(i=0;i<d;i++){
        for(j=0;j<d;j++){
            scanf("%d", &bil2[i][j]);}}
    for(i=0;i<d;i++){
        for(j=0;j<d;j++){
            for(c=0;c<d;c++){
                hasil = hasil+bil1[i][c]*bil2[c][j];}
            bil3[i][j]=hasil;
            hasil=0;}}
    printf("Matriks AXB\n");
    for(i=0;i<d;i++){
        for(j=0;j<d;j++){
            printf("%d ", bil3[i][j]);}
        printf("\n");}
}